cloud/src/meta-service/codec.h

```diff
@@ -27,6 +29,7 @@ struct EncodingTag {
         BYTES_TAG = 0x10,
         NEGATIVE_FIXED_INT_TAG = 0x11,
         POSITIVE_FIXED_INT_TAG = 0x12,
+        VERSIONSTAMP_TAG = 0x13,
```

Comment:
can we reuse the BYTE_TAG?

Reply:
An external `BYTE_ESCAPE` and `ESCAPED_00` will be introduced if the `versionstamp` contains a '0x00'.